src/test/java/org/folio/marc4ld/mapper/field502/Marc2Bibframe502IT.java

```diff
@@ -0,0 +1,83 @@
+package org.folio.marc4ld.mapper.field502;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+import static org.folio.marc4ld.mapper.test.TestUtil.validateEdge;
+
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.InstanceOfAssertFactories;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.marc4ld.mapper.test.SpringTestConfig;
+import org.folio.marc4ld.service.marc2ld.Marc2BibframeMapperImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class Marc2Bibframe502IT {
+
+  @Autowired
+  private Marc2BibframeMapperImpl marc2BibframeMapper;
+
+  @Test
+  void shouldMapField502() {
+    // given
+    var marc = loadResourceAsString("fields/502/marc_502.jsonl");
+
+    // when
+    var result = marc2BibframeMapper.fromMarcJson(marc);
+
+    // then
+    assertThat(result)
+      .isNotNull()
+      .extracting(this::getDissertationEdge)
+      .satisfies(e -> validateEdge(e, PredicateDictionary.DISSERTATION,
+        List.of(ResourceTypeDictionary.DISSERTATION),
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("dissertation label"),
```

Comment:
why not ```LABEL.getValue()```?

Reply:
using part of the code database in tests isn't good practice
It  hides mistakes and doesn't show the expected results

Suggest keeping this practice and don't use classes from code or our main libs